src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,695 @@
+
+## Requirements [#requirements]
+
+Before starting this tutorial, please read the [requirements for automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign) and for [the SCIM API](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management). 
+
+Note that the API does **not** give you the ability to manage access grants (assigning your New Relic users access to accounts and roles). This means that after you're done creating an integration with this API, you must next use the **Organization and access** UI to create access grants. For more information, see [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
+
+## Overview [#overview]
+
+Before using this tutorial, we first recommend reading these docs:
+
+* [Where to find user management features in the UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where)
+* [Important user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)
+* [Our primary SCIM API reference](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management/)
+* Some SCIM 2.0 RFC documents from the Internet Engineering Task Force that are most relevant: [RFC 7643 - SCIM Core Resources and Extensions](https://datatracker.ietf.org/doc/html/rfc7643#section-4), [RFC 7643 - JSON Representation](https://datatracker.ietf.org/doc/html/rfc7643#section-8), and [RFC 7644 - SCIM Protocol](https://datatracker.ietf.org/doc/html/rfc7644).
+
+This tutorial walks you through how to:
+
+* [Configure your authentication domain for SCIM](#configure-scim)
+* [Create users and user groups in your system](#create-in-idp)
+* [Connect to the SCIM API](#connect)
+* [Create users in your authentication domain](#create-users)
+* [Create groups in your authentication domain](#create-groups)
+* [View users and groups in your authentication domain](#view)
+* [Update a user's attributes](#update-user)
+* [Update a group's members](#update-group)
+* [Delete users and groups](#delete)
+* [Manage user type](#manage-user-type)
+
+## Configure your authentication domain for SCIM [#configure-scim]
+
+Before you can use the SCIM API, you must first [enable SCIM for your authentication domain](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#source-users). Note that the API access token is displayed only once after you save the configuration, so save it somewhere safe for later user. 
```

Comment:
I think this tutorial should include the minimum amount of information needed to do a thing, instead of linking out to a more complete doc. This doesn't feel like the minimum amount of info here.